core/domain/feedback_jobs_continuous_test.py

```diff
@@ -147,16 +147,17 @@ def test_multiple_threads_multiple_exp(self):
             ModifiedFeedbackAnalyticsAggregator.start_computation()
             self.assertEqual(self.count_jobs_in_taskqueue(), 1)
             self.process_and_flush_pending_tasks()
+            feedback_analytics = (
```

Comment:
I agree with @seanlip. I think you should still have a function get_thread_analytics() that calls get_thread_analytics_multi() with 1 element, and not change the existing tests. It seems like getting analytics for only 1 exploration is a relatively common occurrence, and `get_thread_analytics_multi([exp_id])[0]` seems much more clunky than `get_thread_analytics(exp_id)`.

Also, add a test that actually gets more than one exp_id.